src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/ExtensionMemberImportCompletionHelper.SymbolComputer.cs

```diff
@@ -78,32 +81,43 @@ public static async ValueTask UpdateCacheAsync(Project project, CancellationToke
                 await SymbolTreeInfo.GetInfoForMetadataReferenceAsync(project.Solution, peReference, checksum: null, cancellationToken).ConfigureAwait(false);
         }
 
-        public async Task<ImmutableArray<IMethodSymbol>> GetExtensionMethodSymbolsAsync(bool forceCacheCreation, bool hideAdvancedMembers, CancellationToken cancellationToken)
+        public async Task<ImmutableArray<ISymbol>> GetExtensionMemberSymbolsAsync(
+            bool forceCacheCreation, bool hideAdvancedMembers, bool isStatic, CancellationToken cancellationToken)
         {
             try
             {
+                // Enable if, during debugging, you want to be able to easily debug into the PE reference processing
```

Comment:
nit: FWIW, in the compiler, instead of ifdef we just avoid the parallelism when debugging. 
See `MethodCompiler` checking `ConcurrentBuild`; our unittests set `ConcurrentBuild` to `false` when debugger is attached. 
See `CSharpTestBase.CreateCompilationCore` (line 1836)

Reply:
i'll look into that.